pokedex_async_redux/lib/home_page.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: Scaffold(
+        body: Center(
+          child: Text('Home Page'),
```

Comment:
kindly add a todo comment that this will be placed in a constant later 🙂

Reply:
done po sir!